code/Test_definitions/Geofencing.feature

```diff
@@ -0,0 +1,114 @@
+
+
+@Geofencing 
+Feature: Camara Geofencing Subscriptions API ,0.2.0-rc3 Operations on subscriptions 
```

Comment:
Target version for geofencing is v0.3.0, right @maxl2287?

Reply:
@jlurien Correct.